pipelined/bevy_pbr2/src/material.rs

```diff
@@ -112,6 +112,13 @@ pub struct StandardMaterialUniformData {
     /// defaults to 0.5 which is mapped to 4% reflectance in the shader
     pub reflectance: f32,
     pub flags: u32,
+    /// When the alpha mode mask flag is set, any base color alpha above this cutoff means fully opaque,
+    /// and any below means fully transparent.
+    pub alpha_cutoff: f32,
+    // NOTE: Pad up to 16-byte alignment for WGSL
+    pub _padding0: f32,
```

Comment:
Yes. It should. I wouldn't have put this here if it wasn't needed. I think maybe it's the 'it doesn't pad at the end of a struct' thing. But maybe that is fixed since the rewrite. I should try removing them now...

Reply:
Yup, they aren't needed anymore.